x-pack/plugins/security_solution/server/lib/risk_engine/utils/transforms.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TransformGetTransformStatsResponse } from '@elastic/elasticsearch/lib/api/types';
+import { elasticsearchServiceMock } from '@kbn/core/server/mocks';
+import { scheduleTransformNow } from './transforms';
+
+const transformId = 'test_transform_id';
+
+const startedTransformsMock = {
+  count: 1,
+  transforms: [
+    {
+      id: 'test_transform_id_1',
+      state: 'started',
+    },
+  ],
+} as unknown as TransformGetTransformStatsResponse;
```

Comment:
It doesn't appear that the `as unknown` is necessary here: 
```suggestion
} as TransformGetTransformStatsResponse;
```

ditto below.